overrides-c9s.yaml

```diff
@@ -6,6 +6,10 @@
 repos:
  - c9s-baseos-mirror
  - c9s-appstream-mirror
+# rt-mirror is needed while building scos node images through CI in case the composes have already moved kernel version
+# this is true for any othe package as well - so we will need to find a better solution for this [TODO]
+ - c9s-rt-mirror
+ - c9s-nfv-mirror
```

Comment:
@jlebon is there a better alternative to keep the building of the node image in sync with the bootimage? how  is it done for RHCOS?

Reply:
should SCOS just build from the mirror rather than the composes?

Reply:
It's awkward but currently, the base image is built with the pungi compose _and_ mirror (see convo in https://github.com/openshift/os/pull/1765), so yeah we should just match this for the extensions too.

But these should be in `extensions-okd-c9s.yaml`, not here. (And only in the `kernel-rt` extension since the repo doesn't exist on every arch.)

Reply:
makes sense. moved it to the extensions file.